common/emotes.go

```diff
@@ -17,19 +17,11 @@ var (
 	reWrappedEmotes = regexp.MustCompile(`[:\[][^\s:\/\\\?=#\]\[]+[:\]]`)
 )
 
-func init() {
-	Emotes = NewEmotesMap()
-}
-
-func NewEmotesMap() EmotesMap {
```

Comment:
Removing this stuff is fine, but you have to actually initialize the map if you do.  Loading emotes panics currently.
![image](https://user-images.githubusercontent.com/6540297/163678977-e1bfa35e-4171-4c18-8730-2be8807acdc0.png)

Reply:
fixed in 0586c8076dddc78a4c2d890fb666275216fb05e4